src/plugin/helpers/prosemirror.ts

```diff
@@ -126,47 +133,92 @@ const moveNode = (consumerPredicate: Predicate) => (
   return true;
 };
 
-const moveNodeUp = (predicate: Predicate) => (
-  getPos: () => number | undefined
-) => (view: EditorView, run = true) =>
-  moveNode(predicate)(getPos, view.state, run && view.dispatch, "up");
+const moveNodeUp = (
+  predicate: Predicate,
+  sendTelemetryEvent: SendTelemetryEvent
+) => (getPos: () => number | undefined) => (view: EditorView, run = true) => {
+  const sendEvent = () =>
+    sendTelemetryEvent?.(CommandTelemetryType.PMEUpButtonPressed);
```

Comment:
Reckon it's worth having a separate event for 'top' and 'bottom' presses – be really interested to know if they're used!